settings/deno.vim

```diff
@@ -1,11 +1,18 @@
 function! Vim_lsp_settings_deno_get_blocklist() abort
+    " Deno project
     if !empty(lsp#utils#find_nearest_parent_file(lsp#utils#get_buffer_path(), 'deno.json'))
         return []
     endif
     if !empty(lsp#utils#find_nearest_parent_file(lsp#utils#get_buffer_path(), 'deno.jsonc'))
         return []
     endif
 
+    " Deno with package.json, create directory 'node_modules/.deno/' and some module under it.
+    if !empty(lsp#utils#find_nearest_parent_file_directory(lsp#utils#get_buffer_path(), 'node_modules/.deno/'))
+        return []
+    endif
+
+    " Is not non-Deno(=npm) project
```

Comment:
Nit picking.
Is not Deno(=npm) project
or
Is non-Deno(=npm) project

Reply:
Thanks

But this statement is `empty` check.

Result: npm project mark(`node_modules`) is not exists.

Reply:
Thanks. Undestood.
`not non` is bit a confusing to me 😅

Reply:
If there is a better comment, I will adopt it, and/or if it is misleading, I will remove it.